trpcultivate_phenotypes/src/TripalCultivateValidator/ValidatorTraits/FileTypes.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Drupal\trpcultivate_phenotypes\TripalCultivateValidator\ValidatorTraits;
+
+/**
+ * Provides setters focused for setting a file types and file media type (MIME) 
+ * supported by the importer, and getter to retrieve the set value.
+ */
+trait FileTypes {
+  /**
+   * The key used by the setter method to create a file types element 
+   * in the context array, as well as the key used by the getter method 
+   * to reference and retrieve the file types element value.
+   * 
+   * @var string
+   */
+  private string $trait_key = 'file_types';
+
+  /**
+   * Sets a file types with the file media type.
```

Comment:
This wording is not very clear to me. What about this?

```suggestion
   * Sets the file extensions and associated mime-types that a validator checking the
   * file metadata should ensure the file is one of.
```

Reply:
No longer part of the source code.